ruby-challenges.rb

```diff
@@ -32,3 +62,23 @@
 # Expected output: 'LEARN is not a palindrome'
 palindrome_tester3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+#PSEUDOCODE
+# Create method named is_palindrome
+# input: string
+# use downcase on string, otherwise conditional will not recognize upcase as the same letter
+# save downcase string to variable lowercase_string
+# use conditional to check if lowercase_string is equal to lowercase_string.reversed
+# puts concatenated string stating it is a palindrome, and vice versa if not.
+# output: string is/is not a palindrome
+
+def is_palindrome(string)
+    lowercase_string = string.downcase
+    if lowercase_string == lowercase_string.reverse
+        "#{lowercase_string} is a palindrome"
+    else
+        "#{lowercase_string} is not a palindrome"
+    end
+end
+
+p is_palindrome(palindrome_tester1)
```

Comment:
Great work keeping the cases of the letters in mind! Just one thing, the string interpolation is returning the lowercase word instead of the original word!